notification-portlet-webcomponents/notification-icon/src/components/NotificationItem.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <dropdown-item :target="target" :href="url" @click="onClickAction()">{{notification.body}}</dropdown-item>
```

Comment:
This could use some additional styling when the item is read vs unread.
The `isRead` detection is already in place on line 19.